datagateway_api/common/config.py

```diff
@@ -1,102 +1,81 @@
-from enum import Enum
 import json
 import logging
 from pathlib import Path
 import sys
+from typing import Optional
 
-import requests
+from pydantic import (
+    BaseModel,
+    StrictBool,
+    StrictInt,
+    StrictStr,
+    ValidationError,
+    validator,
+)
 
 
 log = logging.getLogger()
 
 
-class APIConfigOptions(Enum):
+class DatagatewayAPI(BaseModel):
```

Comment:
I don't want to pick _that_ picky reviewer, but could you change this (and the two other instances of it in this file) to be `DataGatewayAPI` please?